params/config.go

```diff
@@ -220,6 +220,9 @@ type UpgradeConfig struct {
 	// forks must be present or upgradeBytes will be rejected.
 	NetworkUpgrades *NetworkUpgrades `json:"networkUpgrades,omitempty"`
 
+	// Config for modifying state as a network upgrade.
+	StateUpgrades []StateUpgrade `json:"stateUpgrades,omitempty"`
```

Comment:
can we make this a `map[uint64]StateUpgrade` so that StateUpgrades do not need to specify a timestamp.

Reply:
We could do this if you strongly preferred, it would require changing the code and adding some sort logic which would be different than what we have for precompiles, I'd slightly prefer to keep the symmetry.

Reply:
Good point, will leave this as is.